File: Python/Dice Cup.py
inpt = input().split()

N, M = int(inpt[0]), int(inpt[1])

nrs_list = [N, M]

if N == M:
  print(M+1)
else:
  for i in range(min(nrs_list)+1, max(nrs_list)+2, 1):
    print(i)

File: C#/Dice Cup.cs
using System;
using System.Collections.Generic;
using System.Linq;

    class Program
    {
        static void Main(string[] args)
        {
            string[] input = Console.ReadLine().Split(' ');

            int N = int.Parse(input[0]);
            int M = int.Parse(input[1]);

            List<int> nrs = new List<int>();
            nrs.Add(N);
            nrs.Add(M);

            if (N == M)
            {
                Console.WriteLine(M + 1);
            }
            else
            {
                for (int i = nrs.Min()+1; i < nrs.Max()+2; i++)
                {
                    Console.WriteLine(i);
                }
            }

            Console.ReadLine();
        }

    }
